# class Outer:
#     def outer_method(self):
#         print('this is inner')
#     class Inner:
#         def inner_method(self):
#             print('this is inner')

# o1=Outer()
# inObj=o1.Inner()
# inObj.inner_method()


# try again

# try:
#     variable = 8
#     print(variable + "hello")
#     print(variable / 4)

# except ZeroDivisionError as e:
#      print("Divided by zero")
# except (ValueError, TypeError):
#      print("Error occurred")










